.ci-mgmt.yaml

```
@@ -30,3 +30,13 @@ runner:
   prerequisites: pulumi-ubuntu-8core
   buildSdk: pulumi-ubuntu-8core
   publish: pulumi-ubuntu-8core
+actions:
+  preTest:
+
+    - name: make upstream
+      run: |
+        make upstream
```

Optional nit: This is equivalent.
```suggestion
    - run: make upstream
```